plugins/mtd/mtd.quirk

```diff
@@ -16,3 +16,8 @@ Flags = no-probe
 [MTD\NAME_BIOS]
 Name = Internal SPI Controller
 #FirmwareGType = FuIfdFirmware
+
+# B&R Industrial Automation GmbH
+[PCI\VEN_1677]
+Flags = usable-during-update,unsigned-payload
```

Comment:
for *all* B&R devices now and in the future? I'd prefer the flags were set with a VEN + DEV.